app/frontend/utils/formio-component-traversal.ts

```diff
@@ -24,13 +23,25 @@ const findPanelComponents = (components) => {
   return panelComponents
 }
 
+export const nestedComponentsIncomplete = (components) => {
```

Comment:
change name to `getNestedComponentsIncomplete`. The original name can be mis understood as a boolean return